dependencies.properties

```diff
@@ -49,8 +49,8 @@ maven.com_google_api_api_common=com.google.api:api-common:1.8.1
 maven.org_threeten_threetenbp=org.threeten:threetenbp:1.4.0
 maven.com_google_api_grpc_grpc_google_iam_v1=com.google.api.grpc:grpc-google-iam-v1:0.13.0
 maven.com_google_api_grpc_proto_google_iam_v1=com.google.api.grpc:proto-google-iam-v1:0.13.0
-maven.com_google_http_client_google_http_client=com.google.http-client:google-http-client:1.34.0
-maven.com_google_http_client_google_http_client_jackson2=com.google.http-client:google-http-client-jackson2:1.34.0
+maven.com_google_http_client_google_http_client=com.google.http-client:google-http-client:1.34.1
```

Comment:
We can't update libraries-bom to 1.34.1 until we push some changes in either protobuf-java or java-core, possibly both. Could we hold off on this one until those are done?

Reply:
java-core has released with http-client 1.34.1

Reply:
The change we need pushed from java-core is actually https://github.com/googleapis/java-core/pull/140. Otherwise errorprone fails the upper bounds check.